airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -338,7 +338,7 @@ def post_dag_run(*, dag_id: str, session: Session = NEW_SESSION) -> APIResponse:
                 run_id=run_id,
                 execution_date=logical_date,
                 data_interval=dag.timetable.infer_manual_data_interval(run_after=logical_date),
-                state=DagRunState.QUEUED,
+                state=post_body.get("state"),
```

Comment:
```suggestion
                state=post_body.get("state", DagRunState.QUEUED),
```

Reply:
I think that the schema validation will already populate the `queued` field in the body with the default value from the openAPI schema. Do you think its necessary to define the default twice for clarity?

Reply:
If the field is always present, this should be `post_body["state"]` instead.

Reply:
My bad, I was misunderstanding how json-schema works:

> The default keyword specifies a default value. This value is not used to fill in missing values during the validation process. Non-validation tools such as documentation generators or form generators may use this value to give hints to users about how to use a value. However, default is typically used to express that if a value is missing, then the value is semantically the same as if the value was present with the default value.

So you were totally right, we should enforce the default here 👍